src/Service.Tests/SqlTests/GraphQLPaginationTests/GraphQLPaginationTestBase.cs

```diff
@@ -886,6 +974,25 @@ public async Task RequestInvalidZeroFirst()
             SqlTestHelper.TestForErrorInGraphQLResponse(result.ToString(), statusCode: $"{DataApiBuilderException.SubStatusCodes.BadRequest}");
         }
 
+        /// <summary>
+        /// Request an invalid number of entries for a pagination page
+        /// </summary>
+        [TestMethod]
+        public async Task RequestInvalidMaxSize()
```

Comment:
do we also want a test that validates `100000` explicitly renders a request successful? that would be "regression detection" test because if someone inadvertently changes maxpagesize from a value of 100000, we'd catch that before allowing a  breaking change to be committed.

Reply:
Your tests `RequestMaxUsingNegativeOne()` only ensures that `-1` translates into whatever value happens to be set as maxpagesize.

Reply:
Yes let me add this test.

Reply:
Added a comment as discussed on call in configvalidator with note that changing these values could be a breaking change.